conans/client/installer.py

```diff
@@ -462,7 +462,7 @@ def _handle_node_editable(self, node, profile_host, profile_build, graph_lock):
         # Get source of information
         conanfile = node.conanfile
         ref = node.ref
-        package_layout = self._cache.package_layout(ref)
+        package_layout = self._cache.get_ref_layout(ref)
```

Comment:
@czoido @memsharded should I have used `get_latest_` instead?

Reply:
I think this is wrong, this must be a ``package_folder``, but it seems you are using a ``recipe folder``

Reply:
I am surprised that this is not breaking tests, but ok, I guess too many commented tests now.

Reply:
@memsharded  Would it be fine if add

```python
        package_layout = self._cache.get_pkg_layout(node.pref)
```
?

Reply:
I guess the same, the test is not raising anything because of all the commented tests...

Reply:
Yes, all tests related to editables are commented so changes here will probably not affect. Why were this changes necessary? Do you recall @franramirez688?
Maybe we can divide this PR in several phases so we don't have so many changes at once and we have very clear what the changes in sources (not the tests) are fixing?

Reply:
Ok, yes, that is true, that is editable... I guess we can leave it at the moment, until editables are fixed.

Reply:
Ok then, keeping it for now